cairo/ethereum_rlp/rlp.cairo

```diff
@@ -222,6 +222,52 @@ namespace ExtendedImpl {
         );
         return extended;
     }
+
+    // @notice Converts a Simple to an Extended. This is needed because `decode` returns a Simple but `trie.cairo` types are Extended.
+    // @dev in Python, this conversion is trivial since Simple is included in Extended.
+    //      in Cairo, we need to check the type of the Simple and convert it to the correct Extended variant.
+    //      Perhaps we should use either only Extended or only Simple in the RLP encoding library.
+    func from_simple(simple: Simple) -> Extended {
```

Comment:
also, we can't test this because it's in a namespace?

Reply:
Mhm I think since it's imported in another file, we can test it, let me try

Reply:
Ah no it doesn't work